src/core/tritonserver.h

```diff
@@ -1004,12 +1003,26 @@ TRITONSERVER_EXPORT TRITONSERVER_Error* TRITONSERVER_ServerModelMetadata(
     TRITONSERVER_Server* server, const char* model_name,
     const char* model_version, TRITONSERVER_Message** model_metadata);
 
+/// Get the statistics of the model being served as a TRITONSERVER_Message
+/// object. The caller takes ownership of the object and must call
+/// TRITONSERVER_MessageDelete to release the object.
+/// \param server The inference server object.
+/// \param model_name The name of the model to get statistics for.
+/// \param model_version The version of the model to get readiness for.
```

Comment:
This comment needs updating, no need to run CI just let me know when it is updated.

Reply:
Updated